backend/api/views/main.py

```diff
@@ -22,6 +22,24 @@ def get_persons():
     return create_response(data={"persons": serialize_list(persons)})
 
 
+"""
+Template for different endpoints -> both GET and POST
+To write these, make sure to create a new .py. Add a new blueprint to the file and the __init__.py
+
+# Getting all the information in the specified data table
+# To specify specific data you want to call from the class you specified, call methods accordingly and input parameter: data = {}, status = 200, message="ex: success"
+@main.route("/<yourclassname>", methods=["GET"])
+def get_<yourclassname>():
+    sample_var = <yourclassname>.query.all()
+    return create_response(data={"<yourclassname>": serialize_list(messages)})
+
+@main.route("/<yourclassname>", methods=["POST"])
+def add_<yourclassname>():
+    sample_args = request.args
+    new_data = dataclassname(sample_args ...)
+    return create_response(status=200, message="success")
+"""
+
 # function that is called when you visit /persons
 @main.route("/messages", methods=["GET"])
 def get_messages():
```

Comment:
If you have this function defined in a separate views file and register the blueprint, then I think this function can be removed.